<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteSaleorderDialog</class>
 <widget class="QDialog" name="DeleteSaleorderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxOutput">
     <property name="title">
      <string>Информация по чеку</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTerminal">
        <property name="minimumSize">
         <size>
          <width>59</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAdress">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelShift">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonUpdate">
          <property name="text">
           <string>Удаление</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Другая транзакция</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignLeft">
    <widget class="QGroupBox" name="groupBoxInput">
     <property name="minimumSize">
      <size>
       <width>215</width>
       <height>142</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Искомая транзакция</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonFind">
        <property name="text">
         <string>Поиск транзакции</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>№ смены</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Терминал</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditShift"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditTerminal"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditNumCheck"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>№ чека</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditTerminal</tabstop>
  <tabstop>lineEditShift</tabstop>
  <tabstop>lineEditNumCheck</tabstop>
  <tabstop>pushButtonFind</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
